setup.py

```diff
@@ -1,8 +1,30 @@
 from setuptools import setup, find_packages
 from setuptools.command.install import install
 from setuptools.command.develop import develop
+from setuptools.command.build_py import build_py
 import sys
-import subprocess
+from os.path import join
+from os import walk
+
+STRIP_DIRS = ["src", "tests"]
+
+
+def strip_file(file_path):
+    from strip_hints import strip_file_to_string
+
+    if file_path.endswith('.py'):
+        stripped = strip_file_to_string(file_path, to_empty=True, only_assigns_and_defs=True)
+        print(file_path)
```

Comment:
Is this a debugging print?

Reply:
I was implementing what the strip_hints.sh shell script had.  It prints out each file it processes.  I don't have a strong opinion on them so I'm happy to remove them.

Reply:
The print statements were originally in there just to ensure the actions were being logged in the Travis output. I don't think it hurts to be verbose in this case 🤷‍♂

Reply:
Usually I'm not a fan of too many print statements but in this case I think it's useful.  There aren't very many files and it's nice to see that it's running as expected.